test/parallel/test-fs-timestamp-parsing-error.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+require('../common');
+var assert = require('assert');
+var fs = require('fs');
+
+[undefined, null, {}, []].forEach((input) => throws(input));
```

Comment:
You could probably just inline `throws` like you did in the "does not throw" case.